kubernetes_tests/test_base.py

```diff
@@ -287,12 +289,13 @@ def start_job_in_kubernetes(self, dag_id, host):
         result_json = self.start_dag(dag_id=dag_id, host=host)
         dag_runs = result_json["dag_runs"]
         assert len(dag_runs) > 0
-        logical_date = None
+        run_after = None
```

Comment:
Same, we should not remove `logical_date`.

Reply:
Changed it.